src/Languages/lang.pt-br.php

```diff
@@ -61,252 +62,259 @@
+    'settings_check_perms' => 'Suas configurações não puderam ser salvas, verifique as permissões do seu arquivo
+                            config.yml.',
+    'settings_cannot_write' => 'PHP Censor não pode escrever o arquivo config.yml, as configurações não serão salvas
+                            corretamente  até que isso seja resolvido.',
+    'settings_github_linked' => 'Sua conta do GitHub foi vinculada.',
+    'settings_github_not_linked' => 'Sua conta do GitHub não pode ser vinculada.',
+    'build_settings' => 'Configurações de Compilação',
+    'github_application' => 'Aplicação do GitHub',
+    'github_sign_in' => 'Antes de começar a usar o GitHub, você precisa <a href="%s">se autenticar</a> e permitir que o
+                            PHP Censor acesse a sua conta.',
+    'github_linked' => 'PHP Censor foi vinculado a sua conta do GitHub.',
+    'github_where_to_find' => 'Onde encontrar...',
+    'github_where_help' => 'Se você é dono da aplicação que você gostaria de usar, você pode encontrar essa informação
+                        na <a href="https://github.com/settings/applications">área de configurações da sua
+                        aplicação</a>.',
+
+    'email_settings' => 'Configurações de Email',
+    'email_settings_help' => 'Antes que o PHP Censor possa enviar e-mails com atualizações de estado, você deve
+                            configurar os dados do seu SMTP abaixo.',
+
+    'application_id' => 'ID da Aplicação',
+    'application_secret' => 'Segredo da Aplicação',
+
+    'smtp_server' => 'Servidor SMTP',
+    'smtp_port' => 'Porta SMTP',
+    'smtp_username' => 'Usuárop SMTP',
+    'smtp_password' => 'Senha SMTP',
+    'from_email_address' => 'Endereço de E-mail',
+    'default_notification_address' => 'Endereço de E-mail Padrão para Notificações',
+    'use_smtp_encryption' => 'Usar criptografia SMTP?',
+    'none' => 'Nenhum',
     'ssl' => 'SSL',
     'tls' => 'TLS',
 
-    'failed_after' => 'Consider a build failed after',
-    '5_mins' => '5 Minutes',
-    '15_mins' => '15 Minutes',
-    '30_mins' => '30 Minutes',
-    '1_hour' => '1 Hour',
-    '3_hours' => '3 Hours',
+    'failed_after' => 'Considere uma compilação como fracassada após',
+    '5_mins' => '5 Minutos',
+    '15_mins' => '15 Minutos',
+    '30_mins' => '30 Minutos',
+    '1_hour' => '1 Hora',
+    '3_hours' => '3 Horas',
 
     // Plugins
-    'cannot_update_composer' => 'PHP Censor cannot update composer.json for you as it is not writable.',
-    'x_has_been_removed' => '%s has been removed.',
-    'x_has_been_added' => '%s has been added to composer.json for you and will be installed next time
-                            you run composer update.',
+    'cannot_update_composer' => 'PHP Censor não pode atualizar o composer.json por você pois não pois não é gravável.',
+    'x_has_been_removed' => '%s foi removido.',
+    'x_has_been_added' => '%s foi adicionado ao composer.json por você e será instalado da próxima vez que você
```

Comment:
"for you" -> "para você" ?
I'm not sure about the context, though. "Por você" may be better in some cases.